lib/util.js

```diff
@@ -506,6 +506,7 @@ const util = {
 
     const args = util.buildArgsToString(config.buildArgs())
     util.run('gn', ['gen', config.outputDir, '--args="' + args + '"'], options)
+    util.run('gn', ['check', config.outputDir, '//brave/*'], options)
```

Comment:
I don't think it makes sense to run this as part of the build, that's too much wasted time